docs/gno-infra/setting-up-a-faucet.md

```diff
@@ -0,0 +1,173 @@
+---
+id: setting-up-a-faucet
+---
+
+# Setting up a faucet for your Gno network
+
+In this tutorial, we will cover how to run a local native currency faucet that 
+works seamlessly with a Gno node. Using the faucet, any address can get a hold
+of testnet GNOTs.
+
+## Prerequisites
+- Git
+- Go 1.21+
+- Make (for running Makefiles)
+- `gnoland` & `gnokey` installed
+- A Gno.land keypair generated using [`gnokey`](../gno-tooling/cli/gnokey.md)
+
+## Premining funds to an address
```

Comment:
We should drop this section completely, and not reference `genesis_balances.txt` anywhere -- we have adequate `genesis balances` commands for premining accounts